File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
          unordered_map<int, int> hm;
  for (int i = 0; i < nums.size(); i++) {
    if (hm.find(target - nums[i]) != hm.end()) {
      auto res = vector<int>{i, hm[target - nums[i]]};
      sort(res.begin(), res.end());
      return res;
    }
    hm[nums[i]] = i;
  }
  return vector<int>{};

    }
};

File: Data Structures & Algorithms/length-of-last-word/submission-0.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        n = len(s)
        res = 0
        for i in range(n-1, -1, -1):

            if s[i] != " ":
                res += 1
            if i > 0 and s[i] != " " and s[i-1] == " ":
                break
        return res
            